code/modules/antagonists/wizard/equipment/spellbook_entries/_entry.dm

```diff
@@ -203,6 +203,12 @@
 		log_spellbook("[key_name(user)] bought [src] for [cost] points")
 		SSblackbox.record_feedback("tally", "wizard_spell_learned", 1, name)
 		log_purchase(user.key)
+
+	ADD_TRAIT(spawned_path, TRAIT_CONTRABAND, INNATE_TRAIT)
+	if(spawned_path.contents)
+		for(var/obj/contained as anything in spawned_path.contents)
+			ADD_TRAIT(contained, TRAIT_CONTRABAND, INNATE_TRAIT)
```

Comment:
```suggestion
	for(var/obj/contained as anything in spawned_path.contents)
		ADD_TRAIT(contained, TRAIT_CONTRABAND, INNATE_TRAIT)
```
`for(var/thing in empty_list)` doesn't do anything anyways. Plus I think this `if` will never fail because it's always a list (which is truthy)